config/puma.rb

```diff
@@ -1,47 +1,17 @@
-# Puma can serve each request in a thread from an internal thread pool.
-# The `threads` method setting takes two numbers a minimum and maximum.
-# Any libraries that use thread pools should be configured to match
-# the maximum value specified for Puma. Default is set to 5 threads for minimum
-# and maximum, this matches the default thread size of Active Record.
-#
+# puma can serve each request in a thread from an internal thread pool.
+# the `threads` method setting takes two numbers a minimum and maximum.
+# any libraries that use thread pools should be configured to match
+# the maximum value specified for puma. default is set to 5 threads for minimum
+# and maximum, this matches the default thread size of active record.
 threads_count = ENV.fetch('RAILS_MAX_THREADS') { 5 }.to_i
-threads threads_count, threads_count
+threads(threads_count, threads_count)
 
-# Specifies the `port` that Puma will listen on to receive requests, default is 3000.
-#
-port        ENV.fetch('PORT') { 3000 }
+# bind the server to a specific uri
+port = ENV.fetch('PORT') { 3000 }
+bind("tcp://0.0.0.0:#{port}")
```

Comment:
Does Heroku bind to `0.0.0.0`?
Is this what you want?  If so that is awesome.

Reply:
we'll, i'm not sure about heroku. i set the host to `0.0.0.0` so that the containers could communicate with each other over the docker [network](https://docs.docker.com/network/) that `docker-compose` adds by default.

when the application is deployed to eric's machine, would we still want our app to bind to `0.0.0.0:<port>` so that it would be visible to the outside world?